File: lec02/lec02q6.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main(){
    int n;
    cin >> n;
    int rem;

    int ans=0;

    while (n>0){
        rem= n%10;
        n=n/10;
        int count;
        ans= ans + count* pow(10, rem);
        count++;
    }
    cout << ans;
}

File: lec02/lec02q6.java
import java.util.*;
class lec02q6{
    public static void main(String[] args){
        Scanner scn= new Scanner(System.in);
        int n= scn.nextInt();

        int rem=0;
        int ans=0;
        int count =0;
        

        while(n>0){
            rem= n%10;
            n=n/10;
            ans+= count* Math.pow(10, rem);
            count++;
        }

        System.out.println(ans);
    }
}